state/memcached/memcached_test.go

```diff
@@ -65,3 +66,18 @@ func TestMemcachedMetadata(t *testing.T) {
 		assert.Equal(t, 5000*time.Millisecond, metadata.timeout)
 	})
 }
+
+func TestMemcachedPing(t *testing.T) {
+	t.Run("ping with correct host", func(t *testing.T) {
```

Comment:
Please complete with success and failure tests.

Reply:
Ok